src/openms/source/APPLICATIONS/TOPPBase.cpp

```diff
@@ -2586,6 +2587,7 @@ namespace OpenMS
     for (int i = argc - 1; i > 0; --i)
     {
       String arg = argv[i];
+      if (arg.hasPrefix("--")) arg = arg.substr(1); // replace '--' by '-' such that it's found in param_map
```

Comment:
but doesnt this now basically mean that for *every* parameter we support `-` and `--` at the same time? basically you could now write `--help` and `-help` with the exact same effect?